source/includes/atlas-sample-data-note.rst

```diff
@@ -0,0 +1,8 @@
+.. admonition:: Try Compass with Sample Data
+   :class: note
+
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`__  
+   is a cloud-hosted database-as-a-service that 
+   provides free :atlas:`sample datasets </sample-data/>` for its 
```

Comment:
"its clusters" reads a little strangely to me since it kind of takes the control away from the user - like "these are Atlas' clusters, not yours".  Maybe replace "its" with "your"?

Reply:
Totally agree, I was struggling with "its." Done.